### PR TITLE
docs(cdk-experimental/menu): add examples to menu documentation

### PR DESCRIPTION
**Pull request by andy9775**
I'm not 100% sure if I connected the examples with menu.md correctly. PTAL.

### CODE REVIEW

File: src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.cdk-menu {
```
Comment (jelbourn) [inline src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css:1]: For the examples, we try to prefix all of the styles with `.example-` so it's clear where they're coming from

File: src/components-examples/cdk-experimental/menu/cdk-menu-menubar/cdk-menu-menubar-example.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  flex-direction: column;
+  min-width: 180px;
+  max-width: 280px;
+  background-color: white;
+  padding: 6px 0;
+}
+
+.cdk-menu hr {
+  width: 100%;
+  color: #0000001f;
```
Comment (jelbourn) [inline src/components-examples/cdk-experimental/menu/cdk-menu-menubar/cdk-menu-menubar-example.css:30]: I don't think the alpha notation here works in older browsers

File: src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css (hunk 1 above)
Comment (andy9775) [reply src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css:1]: Those are the classes that the cdk directives add.

**Comment by andy9775** [comment]
@jelbourn feedback should be addressed.

File: src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css (hunk 1 above)
Comment (jelbourn) [reply src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css:1]: Yeah, we can either add additional class in the example, or put all of these as child selectors, e.g.
```css
.example-menu { ... }
.example-menuitem { ...}

/** OR **/
.example-menu .cdk-menu {
  ...
}

.example-menu .cdk-menuitem {
  ...
}

```

Comment (andy9775) [reply src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css:1]: Ah, ok. I see what you were saying now.

Comment (andy9775) [reply src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css]: @jelbourn Done

**Review by jelbourn — APPROVED** [review]
LGTM

(I was on vacation for a bit)

**Comment by andy9775** [comment]
@jelbourn no problem.

Hope your vacation was good.

**Pull request merged**